docs/content/direct/hub-chart.md

```diff
@@ -0,0 +1,229 @@
+
+```yaml
+WDSes: # all the CPs in this list will execute the wds.yaml PCH
+  - name: <wds1>     # mandatory name of the control plane
+    type: <host|k8s> # optional type of control plane host or k8s (default to k8s, if not specified)
+    APIGroups: ""    # optional string holding a comma-separated list of APIGroups
+    ITSName: <its1>  # optional name of the ITS control plane, this MUST be specified if more than one ITS exists at the moment the WDS PCH starts
+  - name: <wds2>     # mandatory name of the control plane
+    type: <host|k8s> # optional type of control plane host or k8s (default to k8s, if not specified)
+    APIGroups: ""    # optional string holding a comma-separated list of APIGroups
+    ITSName: <its2>  # optional name of the ITS control plane, this MUST be specified if more than one ITS exists at the moment the WDS PCH starts
+  ...
+```
+
+where `name` must specify a unique name of the control plane (not that this must be unique among both ITSes and WDSes), the optional `type` can be either k8s (default) or host, see [here](https://github.com/kubestellar/kubeflex/blob/main/docs/users.md) for more information, the optional `APIGroups` provides a list of APIGroups, see [here](https://docs.kubestellar.io/release-0.22.0/direct/examples/#scenario-2-using-the-hosting-cluster-as-wds-to-deploy-a-custom-resource) for more information, and `ITSName` specify the ITS connected to the new WDS being created (this parameter MUST specified only if more that one ITS exists in the cluster, if no value is specified and only one ITS exists in the cluster, then it will automatically selected).
+
+## KubeStellar Hub Chart usage
+
+A specific version of the KubeStellar hub chart can be simply installed in an existing cluster using the following command:
+
+```shell
+helm upgrade --install ks-hub oci://ghcr.io/kubestellar/kubestellar/hub-chart --version $KUBESTELLAR_VERSION
+```
+
+The above command will install KubeFlex and the Post Create Hooks, but no Control Planes.
+Please remeber to add `--set "kubeflex-operator.isOpenShift=true"`, when installing into an OpenShift cluster.
+
+User defined control planes can be added using additional value files of `--set` arguments, _e.g._:
+
+- add a single ITS named its1 of default vcluster type: `--set-json='ITSes=[{"name":"its1"}]'`
+- add two ITSes named its1 and its2 of of type vlcuster and host, respectevely: `--set-json='ITSes=[{"name":"its1"},{"name":"its2","type":"host"}]'`
+- add a single WDS named wds1 of default k8s type: `--set-json='WDSes=[{"name":"wds1"}]'`
+
+A KubeStellar Hub installation compatible with the common setup suitable for Common Setup described in the [examples](./examples.md) could be achieved with the following command:
+
+```shell
+helm upgrade --install ks-hub oci://ghcr.io/kubestellar/kubestellar/hub-chart --version $KUBESTELLAR_VERSION \
+  --set-json='ITSes=[{"name":"its1"}]' \
+  --set-json='WDSes=[{"name":"wds1"}]'

```

Comment:
Hooray! This is much simpler.

Reply:
yes, in general but in reality for a given Cp of known name and type, it's a single command